configuration/impl/src/main/java/org/jboss/forge/addon/configuration/proxy/ProxySettings.java

```diff
@@ -56,12 +56,7 @@ public static ProxySettings fromForgeConfiguration(Configuration configuration)
       Configuration proxyConfig = configuration.subset("proxy");
       if (proxyConfig != null && !proxyConfig.isEmpty())
       {
-         final String nonProxyHosts = proxyConfig.getString(PROXY_CONFIG_NON_PROXY_HOSTS);
-         List<String> nonProxyHostsList = null;
-         if (nonProxyHosts != null)
-         {
-            nonProxyHostsList = Arrays.asList(nonProxyHosts.split(","));
-         }
+         List<String> nonProxyHostsList = Arrays.asList(proxyConfig.getStringArray(PROXY_CONFIG_NON_PROXY_HOSTS));
```

Comment:
With this change `nonProxyHostsList` will always be not-null. Is that ok?

Reply:
Yes, ForgeProxySelector will check based on contains(host) (line 85/86)